uwblueprint/project-read-frontend
# map enrolment fields to FamilyTable

### Notion ticket link



[Current enrolment section](https://www.notion.so/uwblueprintexecs/Current-enrolment-section-c4f6505ca371412eb09a6b3368294992) note: the edit functionality will be implemented later



### Implementation description

- mapped out enrolments fields to show up in the familyTable (sidebar will come in another ticket)



### Steps to test

1. run the backend and create dummy data
2. see if the fields show up in the family table (there should be 3 fields)



### What should reviewers focus on?

- TS best practices

### Checklist
